vagrant_opm_user/bootstrap.sh

```diff
@@ -20,9 +20,9 @@ sudo apt-get update -y
 
 # OPM packages
 sudo apt-get install libopm-core1-bin -y
-sudo apt-get install libopm-autodiff1-bin -y
-sudo apt-get install libopm-polymer1-bin -y
+sudo apt-get install libopm-simulators1-bin -y
 sudo apt-get install libopm-upscaling1-bin -y
+sudo apt-get install openmpi-bin
```

Comment:
To see if I understand this: the packaged need mpi but do not depend on any one package with it since there are multiple possible choices?